package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-cosmosdb",
-    "version": "0.16.0",
+    "version": "0.16.1-alpha",
```

Comment:
There's another one on line 8. You can manually update both, but often I prefer to just run `npm install` and let it do it for me